notificationworkerlambda/src/main/scala/com/gu/notifications/worker/SenderRequestHandler.scala

```diff
@@ -34,16 +34,27 @@ trait SenderRequestHandler[C <: DeliveryClient] extends Logging {
   implicit val timer: Timer[IO] = IO.timer(ec)
   implicit val logger: Logger = LoggerFactory.getLogger(this.getClass)
 
-  def reportSuccesses[C <: DeliveryClient](notificationId: UUID, range: ShardRange, start: Instant): Pipe[IO, Either[DeliveryException, DeliverySuccess], Unit] = { input =>
-    val notificationLog = s"(notification: ${notificationId} ${range})"
+  def reportSuccesses[C <: DeliveryClient](notificationId: UUID, range: ShardRange, sentTime: Long): Pipe[IO, Either[DeliveryException, DeliverySuccess], Unit] = { input =>
+    val notificationLog = s"(notification: $notificationId $range)"
+    val start = Instant.ofEpochMilli(sentTime)
     val end = Instant.now
     val logFields = Map(
+      "_aws" -> Map(
+        "Timestamp" -> end.toEpochMilli,
+        "CloudWatchMetrics" -> List(Map(
+          "Namespace" -> s"Notifications/${env.stage}/workers",
+          "Dimensions" -> List(List("platform")),
```

Comment:
I think it may be useful when we need to look at the performance of a particular breaking news notification?  We may show  metrics for breaking news only, and we could associate the metric with a particular breaking news notification based on the time.  Please ignore me if cloudwatch metric should not be used for this purpose.

Reply:
Thanks @waisingyiu :) If I recall correctly, providing the notificationId as a metric dimension would incur additional cost (ref: https://docs.google.com/document/d/10AnOZ4MLjuTO7mXaySoVO2SwmroLmns4gGmsfY7egmw/edit#heading=h.ajh00ba9hm68). I think if we needed to analyse the performance of a specific notificationId we'd need to rely on our logs+kibana dashboard (I could've misinterpreted this though!). 

For the metrics, I was hoping to generate a minimum (static) subset of dimensions that would allow us to analyse performance/trends. Based on your response I think there would be value in providing an additional dimension for the sender lambda metrics:
- `platform`: ios or android
- `type`: breakingNews or other

Reply:
> I think if we needed to analyse the performance of a specific notificationId we'd need to rely on our logs+kibana dashboard (I could've misinterpreted this though!).

I think this is the correct approach 👍 

I also agree that adding `platform` and `type` as dimensions would give us some extra benefits without inflating costs too much.